### PR TITLE
update publishing rules for 1.10.2

### PR DESCRIPTION
**Pull request by liggitt**
openshift/kubernetes has the 1.10.2 branch merged and the fork repos have the branch created. need the publish bot to start fanning those out, then will bump deps in https://github.com/openshift/origin/pull/19684

### CODE REVIEW

File: publishing-rules.yaml [hunk 1]
```diff
@@ -2,10 +2,14 @@ skip-godeps: true
...
 - destination: kubernetes
   branches:
-  - name: release-1.9.1
+  - name: origin-3.10-kubernetes-1.10.2
     source:
       branch: master
       dir: vendor/k8s.io/kubernetes
+  - name: release-1.9.1
+    source:
+      branch: release-3.9
```
Comment (deads2k) [inline publishing-rules.yaml]: this was bugged before or something?

**Comment by deads2k** [comment]
It seems reasonable enough to me


/lgtm

File: publishing-rules.yaml (hunk 1 above)
Comment (liggitt) [reply publishing-rules.yaml]: Yes.  I don't think the bot is actually using this version at this point (it must not be), but while I was in here…

**Comment by deads2k** [comment]
failure doesn't look related.

**Comment by deads2k** [comment]
merged to free the re-rebase

**Pull request merged**